src/main/java/uk/gov/ons/ctp/response/casesvc/service/CaseService.java

```diff
@@ -444,37 +444,6 @@ private void processActiveEnrolmentChange(final Case targetCase, final boolean e
         // fetch all B and BI cases associated to the case group being processed
         List<Case> cases =
                 caseRepo.findByCaseGroupFKOrderByCreatedDateTimeDesc(caseGroup.getCaseGroupPK());
-
-        // This section needs to be taken out once activeEnrolment is active
```

Comment:
Hi Adam - Only if statement will need to be removed. Can we also test this in local cluster before review.
we have to follow the smoke test and make sure that the uploaded sample is propagated to actions with proper active enrolment